napari/layers/shapes/_shapes_models/ellipse.py

```diff
@@ -118,3 +120,6 @@ def transform(self, transform):
         self._edge_vertices = centers
         self._edge_offsets = offsets
         self._edge_triangles = triangles
+        self._bounding_box[:, self.dims_displayed] = (
+            self._bounding_box[:, self.dims_displayed] @ transform.T
+        )
```

Comment:
We should probably add tests (unless they already exist) which explicitly make sure that picking shapes that are transformed works as intended

Reply:
will add

Reply:
I have found that we do not have direct transform test.

Reply:
You spot the bug in my code. I have added tests and fixed the code.